apps/AEPSampleApp/extensions/EdgeIdentity.js

```diff
@@ -13,30 +13,30 @@ governing permissions and limitations under the License.
 @format
 */
 
-import React, {Component} from 'react';
-import {Button, StyleSheet, Text, View, ScrollView} from 'react-native';
-import {AEPIdentity} from '@adobe/react-native-aepedgeidentity';
+import React, { Component } from 'react';
+import { Button, StyleSheet, Text, View, ScrollView } from 'react-native';
+import { Identity } from '@adobe/react-native-aepedgeidentity';
```

Comment:
this will be renamed in the edgeidentity branch, let's revert the changes for this file